src/bindings/python/tests/test_torchvision_to_ov/test_preprocessor.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+import copy
+import os
+import pytest
+from PIL import Image
+
+import torch
+import torch.nn.functional as F
+import torchvision.transforms as transforms
+
+from openvino.runtime import Core
+
+from openvino.preprocess.torchvision_to_ov import PreprocessConverter
+
+
+class Convnet(torch.nn.Module):
```

Comment:
You could probably take any model from torchvision instead of creating your own. But it is ok for preprocessing-only tests.

Reply:
Better to play with the toy model in the pre-commit. Real model will spend more time:)